### PR TITLE
Portfolio: Desktop version

### PR DESCRIPTION
**Pull request by praizerema**
Review index.html
Review style.css
Review .gitignore
Review README.md

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,7 +38,36 @@
             />
```
Comment (SajjadAhmad14) [inline index.html:38]: Please add files and folders like `vscode` to `gitignore`.
Please use all the files from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) linter config.

File: style.css [hunk 2]
```diff
@@ -316,3 +317,278 @@ footer {
...
+
+.nav-items {
+  display: none;
+}
+
+.menu-icon {
+  display: block;
+}
+
+@media (min-width: 600px) {
```
Comment (SajjadAhmad14) [inline style.css:339]: Please use the value for min-width property in range of 992px - 1100px.

File: style.css [hunk 3]
```diff
@@ -316,3 +317,278 @@ footer {
...
+    padding: 6% 9%;
+    height: 390px;
+    background-image: url(./images/Illustration-header.png);
+    background-size: 40% 100%;
+    background-repeat: no-repeat;
+    background-position-x: right;
+  }
+}
+
+@media (min-width: 1200px) {
```
Comment (SajjadAhmad14) [inline style.css:596]: Please use the value for min-width property in range of 992px - 1100px.

File: index.html [hunk 4]
```diff
@@ -38,7 +38,36 @@
...
           </svg>
         </div>
-        <div>
+        <div class="nav-items">
```
Comment (SajjadAhmad14) [inline index.html:41]: Please use the font for the navbar from the design page.

File: index.html [hunk 5]
```diff
@@ -164,26 +195,29 @@ <h1 class="nameSalute">Hey There. I’m Praise</h1>
           />
         </div>
       </section>
-      <section class="workSection">
+      <section class="workSection" id="portfolio">
         <h1 class="workTitle">My Recent Works</h1>
```
Comment (SajjadAhmad14) [inline index.html:199]: Please use the font color for the `Recent work` header from the design page.

File: index.html [hunk 6]
```diff
@@ -164,26 +195,29 @@ <h1 class="nameSalute">Hey There. I’m Praise</h1>
...
+              dummy text.
+            </p>
+            <div class="stackList">
+              <div class="stackBox">css</div>
+              <div class="stackBox">html</div>
+              <div class="stackBox">bootstrap</div>
+              <div class="stackBox">Ruby</div>
+            </div>
+            <button class="btn-orange" type="button">See Project</button>
           </div>
```
Comment (SajjadAhmad14) [inline index.html]: Please use the font color for the project title, paragraph, and technologies from the design page.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Praise Ayodele, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (praizerema) [reply index.html:38]: This issue has been resolved

File: style.css (hunk 2 above)
Comment (praizerema) [reply style.css:339]: @SajjadAhmad14  Please explain what you mean here.

**Comment by AshakaE** [comment]
### Hi @praizerema  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the requested changes 👏🏽
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**